src/components/alert/alert.svelte

```diff
@@ -0,0 +1,108 @@
+<svelte:options tag="leo-alert" />
+
+<script lang="ts" context="module">
+  export const types = ['info', 'warning', 'error', 'success'] as const
+  export type AlertType = (typeof types)[number]
+
+  export const modes = ['toast', 'callout'] as const
```

Comment:
It looks like this differs from the current figma, where toast and callback are only differentiated by background color in light mode. Instead there's `simple` and `full` variations that seem to inform the row vs grid layout. I guess this changed recently @aguscruiz ?

Reply:
Nice catch! I've updated this! The colors for dark are actually different with toast/callout too :smile: